gnosis/eth/ethereum_network.py

```diff
@@ -204,6 +204,122 @@ class EthereumNetwork(Enum):
     SHIDEN = 336
     CRONOS_MAINNET = 25
     CRONOS_TESTNET = 338
+    UBIQ_NETWORK_TESTNET = 2
```

Comment:
Other thing that I didn't noticed was the indentation. Should be like the other constants.